Source/WebKit/NetworkProcess/webtransport/cocoa/NetworkTransportSessionCocoa.mm

```diff
@@ -47,20 +50,56 @@
 }
 
 #if HAVE(WEB_TRANSPORT)
-static RetainPtr<nw_parameters_t> createParameters(WebCore::SecurityOriginData&& origin)
+
+static void didReceiveServerTrustChallenge(Ref<NetworkConnectionToWebProcess>&& connectionToWebProcess, URL&& url, WebKit::WebPageProxyIdentifier&& pageID, WebCore::SecurityOriginData&& topOrigin, sec_trust_t trust, sec_protocol_verify_complete_t completion)
+{
+    const char * host = url.host().utf8().data();
```

Comment:
nit: no space before *.  Actually, let's not store host in a const char* because any use of it is a UAF because the CString returned by utf8() is a temporary.